HelpSource/Guides/Internal-Snooping.schelp

```diff
@@ -57,38 +66,39 @@ Server.filenameSymbol.postln; // the path to the file that defined this class
 
 (
 // print all classes whose names start with 'F'
-Class.allClasses.do({ arg class;
-	if (class.name.asString.at(0) == $F, { class.name.postln; });
-})
+Class.allClasses.do { | class |
+	if (class.name.asString.at(0) == $F) { class.name.postln }
+}; "";
```

Comment:
I'll disagree. `|item|` is standard syntax for iterators within the class library. If it's absent from help files, users will be confused when seeing it in the wild.

I don't know which is the right answer. I'm only sure that rigorously avoiding the symbolic notation for arguments is not a totally unproblematic approach.